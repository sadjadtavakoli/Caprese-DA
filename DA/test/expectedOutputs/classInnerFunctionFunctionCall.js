
#3 function TextInlines-test/unit_tests/classinnerfunctionfunctioncall.js-3-17 entered from classInnerFunctionFunctionCall.js-test/unit_tests/classinnerfunctionfunctioncall.js-1-21
#9 function buildInlines-test/unit_tests/classinnerfunctionfunctioncall.js-9-16 entered from classInnerFunctionFunctionCall.js-test/unit_tests/classinnerfunctionfunctioncall.js-1-21
#10 function getTrimmedWidth-test/unit_tests/classinnerfunctionfunctioncall.js-10-12 entered from buildInlines-test/unit_tests/classinnerfunctionfunctioncall.js-9-16
#end Execution