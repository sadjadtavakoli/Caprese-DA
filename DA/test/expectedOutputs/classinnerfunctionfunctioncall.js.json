{"TextInlines-test/unit_tests/classinnerfunctionfunctioncall.js-3-17":{"impacted":["t_1"]},"classinnerfunctionfunctioncall.js-test/unit_tests/classinnerfunctionfunctioncall.js-1-21":{"impacted":["t_7","t_13"]},"buildInlines-test/unit_tests/classinnerfunctionfunctioncall.js-9-16":{"impacted":["t_1","t_19"]},"getTrimmedWidth-test/unit_tests/classinnerfunctionfunctioncall.js-10-12":{"impacted":["t_13"]},"keyMap":{"t_1":"classinnerfunctionfunctioncall.js-test/unit_tests/classinnerfunctionfunctioncall.js-1-21","t_7":"TextInlines-test/unit_tests/classinnerfunctionfunctioncall.js-3-17","t_13":"buildInlines-test/unit_tests/classinnerfunctionfunctioncall.js-9-16","t_19":"getTrimmedWidth-test/unit_tests/classinnerfunctionfunctioncall.js-10-12"}}